VimeoNetworking/VimeoNetworking/NSError+Extensions.swift

```diff
@@ -247,11 +247,8 @@ public extension NSError
         }
         
         for errorJSON in invalidParameters {
-            if let errorCode = errorJSON[self.dynamicType.VimeoErrorCodeKey] as? String {
```

Comment:
Are we sure this doesn't break for any error codes including upload and authentication? Dunno why it would, but I know this code has existed as part of VimeoUpload for a while. cc @alfiehanssen